src/tasks.js

```diff
@@ -6,6 +6,9 @@ export default class Tasks {
   addTask = (description, completed = false, index = this.arrayTasks.length + 1) => {
     if (!this.showDescriptions().find((element) => element === description)) {
       this.arrayTasks.push({ description, completed, index });
+      // for(let i = 0; i < this.arrayTasks.length; i += 1){
+      //   this.arrayTasks[i].completed = false;
+      // }
```

Comment:
- [ ] Please remove commented codes from your codebase. commented codes make your code look messy. kindly remove all commented codes.

Reply:
> I notice that you removed some unused code, that's great !💯 But I think you forgot this part 😅

- [ ] As the previous reviewer mentioned, It would be better to remove unused code from your project rather than comment it out to keep your code clean and follow [the best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)💯

![image](https://user-images.githubusercontent.com/45231276/181750281-0d99a210-25c7-4b54-b455-141811a9c361.png)